x/wasm-storage/keeper/filter.go

```diff
@@ -0,0 +1,121 @@
+package keeper
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/tidwall/gjson"
+)
+
+const (
+	None = iota
+	Mode
+	StdDeviation
+)
```

Comment:
Let's create a type `FilterType`?
```
type FilterType byte

const (
	None FilterType = iota
	Mode
	StdDev
)
```

Reply:
Do we have any use case of the new type Filtertype as of now? If not, we don't have to complicate by introducing another type.

Reply:
I see what you are saying. I converted it to byte so that we can compare the first byte of tally inputs directly to these.